audio_filters/show_response.py

```diff
@@ -8,7 +8,7 @@
 
 
 class FilterType(Protocol):
-    def process(self, sample: float) -> float:
+    def process(self, sample: float) -> float:  # noqa: ARG002
```

Comment:
What would happen if we delete the unused variables?

Reply:
This class is used as interface, so I made this method absctract. Now there are no problems with unused method argument in this place